packages/Uploader/src/Uploader.js

```diff
@@ -164,33 +174,26 @@ const Uploader = React.forwardRef((props, ref) => {
 
   const value = {
     cancelFile,
+    canChooseMultiple,
+    extendedInputProps,
     files,
+    handleChange,
     isCompleted,
     isDisabled,
-    isDragLeave,
     isDraggingOver,
+    isDragLeave,
+    label,
+    maxFileSize,
     onCancel,
     onError,
+    refContainer,
     refInput,
     removeFile,
-    upload,
-    handleChange,
     supportedMimeTypes,
-    canChooseMultiple,
-    refContainer,
-    label,
+    upload,
   };
 
-  const childrenWithProps = React.Children.map(children, child => {
-    return child === null
-      ? null
-      : React.cloneElement(child, {
-          maxFileSize,
-          supportedMimeTypes,
-        });
-  });
-
-  return <UploaderContext.Provider value={value}>{childrenWithProps}</UploaderContext.Provider>;
+  return <UploaderContext.Provider value={value}>{children}</UploaderContext.Provider>;
```

Comment:
moved those 2 props into context, I think there's no reason to put them as props for `<FileList />`. For example, if consumers put different values of `maxFileSize` on both `<Uploader>` and `<Uploader.FileList>`  we don't do anything currently